src/components/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import { useDispatch } from 'react-redux'
+
+import { useAuth } from '../hooks/useAuth'
+import { HomeButton } from './HomeButton'
+import { SignInAndRegister } from './SignInAndRegister'
+import { SignOutButton } from './SignOutButton'
+
+export function Header(): JSX.Element {
```

Comment:
```suggestion
export function Header() {
```
указывать не обязательно